app/mailers/user_notifier.rb

```diff
@@ -12,4 +12,13 @@ def send_post_notification(post)
     mail(:to => recipients, :subject => 'New Request at ' + post.street_1)
   end
 
+  def new_account_notification(user)
```

Comment:
Send the admin email when new account is created. 
Send the user email when his/her account is approved.
(Enter my email for testing reasons... edited in a later commit)